test/unit/friends.saga.test.ts

```diff
@@ -559,4 +563,55 @@ describe('Friends sagas', () => {
       unityMock.verify()
     })
   })
+
+  describe.only('Get Friendship Status', () => {
+    beforeEach(() => {
+      const { store } = buildStore(mockStoreCalls())
+      globalThis.globalStore = store
+    })
+
+    afterEach(() => {
+      sinon.restore()
+      sinon.reset()
+    })
+
+    describe('When the given user id is not a friend and it is not a pending request', () => {
+      it('Should return FriendshipStatus.NONE', () => {
```

Comment:
I would suggest simplifying these blocks into single `it` tests with the whole description since there is only one test per scenario.